### PR TITLE
Fix showdown emoji compression

### PR DESCRIPTION
**Pull request by maxwellito**
Hi everyone,

First, thanks for your library and all the work it represent. Libraries like yours are so good :)

### TL;DR

This PR is just about compressing the Showdown emoji provided in `src/helpers.js`. The file size is 20kb, but once optimised the size is down to .5kb which should be a nice plus for the final dist.

**Tests: OK**

### Long story

I wanted to look a the size of your min dist file to check the size of it. Then when I scrolled I realised that something was repeated many times.

screen shot 2018-02-14 at 19 34 00

It looked like an unoptimised PNG, and that was the case.

After some search I found out that was the Showdown emoji in `src/helpers.js`. So I saved it and optimised it with the latest version of ImageOptim. Then the file size passed from 20kb to .5kb.

**Original file: 20192 bytes**
showdown_emoji_original

**Optimised file: 507 bytes**
showdown_emoji_optim

(I hope GitHub doesn't recompress the files and keep the original files)

Then I updated `helpers.js` and the functional tests.

All seems to work fine.

```
Before:
-rw-r--r--   1 max  staff  191226 14 Feb 20:07 showdown.js
-rw-r--r--   1 max  staff  482834 14 Feb 20:07 showdown.js.map
-rw-r--r--   1 max  staff  106496 14 Feb 20:07 showdown.min.js
-rw-r--r--   1 max  staff   84796 14 Feb 20:07 showdown.min.js.map

After:
-rw-r--r--   1 max  staff  164978 14 Feb 19:58 showdown.js
-rw-r--r--   1 max  staff  454641 14 Feb 19:58 showdown.js.map
-rw-r--r--   1 max  staff   80248 14 Feb 19:58 showdown.min.js
-rw-r--r--   1 max  staff   84795 14 Feb 19:58 showdown.min.js.map
```

Please ask me anything if changes needs to be done, or feels like this PR is not valid.

Thanks,
Max

### CODE REVIEW

**Comment by tivie** [comment]
Thank you very much for this fix.
